config/environments/test.rb

```diff
@@ -8,13 +8,13 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  config.cache_classes = false
-  config.action_view.cache_template_loading = true
+  # Turn false under Spring and add config.action_view.cache_template_loading = true.
+  config.cache_classes = true
```

Comment:
[Auto generated change]
This setting is default in v7.0.x.

https://github.com/rails/rails/blob/d0523fc2c99ffc9899e012fb4c753a7e02b5b974/railties/lib/rails/generators/rails/app/templates/config/environments/test.rb.tt#L11-L12

ref: https://guides.rubyonrails.org/configuring.html#config-cache-classes